docs/core/getting-started/install/win-vs2017.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Install for Windows - Visual Studio 2017
+description: Install for Windows - Visual Studio 2017
+keywords: .NET, .NET Core, Visual Studio 2017, VS2017, C#, Getting Started, Acquisition, Cross Platform
+author: kendrahavens
+ms.author: kendrahavens
+ms.date: 12/19/2016
+ms.topic: article
+ms.prod: .net-core
+ms.devlang: dotnet
+ms.assetid: 
+---
+# Install for Windows - Visual Studio 2017
+## 1. Download Visual Studio 2017 RC
+- An experimental release of .NET Core tools for Visual Studio is included in Visual Studio 2017 RC.
+  ## [Download Visual Studio 2017 RC](https://www.visualstudio.com/vs/visual-studio-2017-rc/)
+## 2. Install the .NET Core tools preview workload
```

Comment:
We have one topic on the vs docs that covers installation. I kind of like how they show the exact steps and screenshots you get:
https://docs.microsoft.com/en-us/visualstudio/install/install-visual-studio

The main difference here would be that in our case we tell them which specifically workload to choose. In this topic, we're going from download VS to install the workload. There are some missing steps in between.